images/go/configs/latest-musl.apko.yaml

```diff
@@ -19,13 +20,8 @@ accounts:
     - username: nonroot
       uid: 65532
       gid: 65532
+  # TODO: this should run as 65532
   run-as: 0
-
 entrypoint:
   command: /usr/bin/go
-
 cmd: help
-
-archs:
```

Comment:
We're going to lose a lot of archs if we do this. Not sure if folks are using them though, without some more querying.

Reply:
Ack. I'm personally fine with it for these images. Happy to hold if we want to wait longer though.